src/main/java/org/bytedeco/javacpp/tools/Generator.java

```diff
@@ -1460,6 +1472,9 @@ boolean classes(boolean handleExceptions, boolean defineAdapters, boolean conver
             out.println("            *env = NULL;");
             out.println("            goto done;");
             out.println("        }");
+            out.println("#if !defined(NO_JNI_DETACH_THREAD) && defined(_WIN32)");
+            out.println("       windowsJniAutoDetachInstance.attached = true;");
```

Comment:
Let's store here the `env` pointer itself instead of just a flag, to be consistent with how it's done with pthreads.

Reply:
... along with something like this?
```java
            out.println("#if !defined(NO_JNI_DETACH_THREAD) && defined(_WIN32)");
            out.println("    if ((*env = windowsJniAutoDetachInstance.env) != NULL) {");
            out.println("        attached = true;");
            out.println("        goto done;");
            out.println("    }");
            out.println("#endif");
```

Reply:
Whoops, missed that short circuit. Good find.